nsmaster/conf/pdns.conf.var

```diff
@@ -1,10 +1,15 @@
 allow-unsigned-notify=yes
 allow-unsigned-supermaster=yes
+api=yes
+api-key=${DESECSTACK_NSMASTER_APIKEY}
 disable-axfr=yes
 setgid=pdns
 setuid=pdns
 slave=yes
 version-string=powerdns
+webserver=yes
+webserver-address=0.0.0.0
```

Comment:
No, the error referred to port 3306 specifically, which is the MySQL port and unrelated to the nsmaster API (port 8081). I realized that nsmaster/dbmaster don't need to run because everything is caught by httpretty. I just verified that removing nsmaster/dbmaster from the test suite fixes the problem. (And it's in line with the pdns-independent test design, so nice!)